readme.html

```diff
@@ -89,13 +97,12 @@ <h2>State file permissions</h2>
   Please check the permissions for the state file as mentioned below.
   <h4>State file path</h4>
   <ul>
-    <li>For Non-NRI instance: /opt/phantom/local_data/app_states/&lt;appid&gt;/&lt;asset_id&gt;_state.json</li>
-    <li>For NRI instance: /&lt;PHANTOM_HOME_DIRECTORY&gt;/local_data/app_states/&lt;appid&gt;/&lt;asset_id&gt;_state.json</li>
+    <li>For Unprivileged install: /&lt;SOAR_HOME_DIRECTORY&gt;/local_data/app_states/&lt;appid&gt;/&lt;asset_id&gt;_state.json</li>
   </ul>
   <h4>State file permissions</h4>
   <ul>
-    <li>File rights: rw-rw-r-- (664) (The phantom user should have read and write access for the state file)</li>
-    <li>File owner: Appropriate phantom user</li>
+    <li>File rights: rw-rw-r-- (664) (The soar user should have read and write access for the state file)</li>
+    <li>File owner: Appropriate soar user</li>
```

Comment:
For lines 100, 104, 105 - did we actually update this in the backend to be soar now instead of phantom?

Reply:
Yes, internally it uses a phantom user not a soar, I have made the changes.